src/components/cookie-banner/cookie-banner.yaml

```diff
@@ -1,4 +1,4 @@
 variants:
 - name: default
   data:
-    content: GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a>
+    html: "GOV.UK uses cookies to make the site simpler. <a href='https://www.gov.uk/help/cookies'>Find out more about cookies</a>"
```

Comment:
Personal preference - I'd invert these quotes so we're using double quotes for attributes consistently.

Reply:
i get this in the macro then:
```
"html": "GOV.UK uses cookies to make the site simpler. <a href=\"https://www.gov.uk/help/cookies\">Find out more about cookies</a>"
```